RELEASE.md

```diff
@@ -11,7 +11,7 @@
 - Fixed bug at `params_as_tensors_for` (#751)
 - Fixed GPflow SciPy optimizer to pass options to _actual_ scipy optimizer correctly (#738)
 - Improved quadrature for likelihoods. Unified quadrature method introduced - `ndiagquad` (#736), (#747)
-- Added support for multi-output GPs, check `multioutput.ipynb` for details (#724)
+- Added support for multi-output GPs, check `multioutput.pct.py` for details (#724)
```

Comment:
Does it make sense to edit history?

Reply:
Probably not, updating old release notes doesn't make much sense, sorry. Reverted